HelloTorchIOS/HelloTorchIOS/LightsLogic.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LightsLogic.swift
+//  HelloTorchIOS
+//
+//  Created by Luis Valle on 08/05/2018.
+//  Copyright © 2018 Novoda. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+class LightsLogic {
+    
+    private let device: AVCaptureDevice? = AVCaptureDevice.default(for: AVMediaType.video)
```

Comment:
for future - if you make a failable initialiser, you could have removed this optionality throughout the class:

```swift
private let device: AVCaptureDevice

init?() {
   guard let device = AVCaptureDevice.default(for: AVMediaType.video) else {
      return nil
   }
   self.device = device
   ...
}
```